pkgs/development/python-modules/ffcv/default.nix

```diff
@@ -27,13 +36,9 @@ buildPythonPackage rec {
   # See https://github.com/libffcv/ffcv/issues/159.
   postPatch = ''
     substituteInPlace setup.py \
```

Comment:
Since this already includes some refactoring, would `removePythonDeps` do the job?

Reply:
So I tried swapping out the `postPatch` for `pythonRemoveDeps` and it kept failing to build.

```
python3.11-ffcv> Running phase: buildPhase
python3.11-ffcv> Executing pypaBuildPhase
python3.11-ffcv> Creating a wheel...
python3.11-ffcv> * Getting build dependencies for wheel...
python3.11-ffcv> * Building wheel...
python3.11-ffcv> ./libffcv/libffcv.cpp: In function ‘void my_fread(int64_t, int64_t, void*, int64_t)’:
python3.11-ffcv> ./libffcv/libffcv.cpp:50:14: warning: ignoring return value of ‘size_t fread(void*, size_t, size_t, FILE*)’ declared with attribute ‘warn_unused_result’ [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result-Wunused-result8;;]
python3.11-ffcv>    50 |         fread(destination, 1, size, (FILE *) fp);
python3.11-ffcv>       |         ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
python3.11-ffcv> Successfully built ffcv-1.0.0-cp311-cp311-linux_x86_64.whl
python3.11-ffcv> Finished creating a wheel...
python3.11-ffcv> Finished executing pypaBuildPhase
python3.11-ffcv> Running phase: pythonRuntimeDepsCheckHook
python3.11-ffcv> Executing pythonRuntimeDepsCheck
python3.11-ffcv> Checking runtime dependencies for ffcv-1.0.0-cp311-cp311-linux_x86_64.whl
python3.11-ffcv>   - fastargs not installed
python3.11-ffcv>   - opencv-python not installed
```

Reply:
This is what I had in there

```
pythonRemoveDeps = [
  "assertpy"
  "fastargs"
  "opencv-python"
  "psutil"
];
```

Reply:
Hm, interesting

Reply:
I only see an `install_requires` in their setup.py, I think `pythonRemoveDeps` should work but we can debug this in another PR